### PR TITLE
update Quick addr

### PR DESCRIPTION
**Pull request by Rishabh42**

### CODE REVIEW

File: truffle-config.js [hunk 1]
```diff
@@ -65,7 +71,7 @@ module.exports = {
         eth_mainnet: {
             provider: () =>
                 new HDWalletProvider(
-                    mnemonic,
+                    this.secrets.mnemonic,
                     "https://mainnet.infura.io/v3/cf38c21326954ac28aa4f8c3ee33550c"
```
Comment (jchittoda) [inline truffle-config.js:75]: Why our infura id is used in the URL ?

File: truffle-config.js [hunk 2]
```diff
@@ -89,7 +95,7 @@ module.exports = {
         eth_kovan: {
             provider: () =>
                 new HDWalletProvider(
-                    mnemonic,
+                    this.secrets.mnemonic,
                     "https://kovan.infura.io/v3/88375992b7cc4e9c81a67c24b2bebdbf"
```
Comment (jchittoda) [inline truffle-config.js:99]: Here as well

File: truffle-config.js [hunk 3]
```diff
@@ -98,16 +104,16 @@ module.exports = {
...
         },
 
-        polygon_mainnet: {
+        mainnet: {
             provider: () =>
                 new HDWalletProvider(
-                    mnemonic,
-                    "https://rpc-mainnet.maticvigil.com/"
+                    this.secrets.mnemonic,
+                    "wss://polygon-mainnet.g.alchemy.com/v2/JO9fRCDkzxzHvjcFPjy5vsFq9UhhRSxh"
```
Comment (jchittoda) [inline truffle-config.js:111]: here also

**Review by jchittoda — COMMENTED** [review]
pls fix the comments

File: truffle-config.js (hunk 1 above)
Comment (Rishabh42) [reply truffle-config.js:75]: That was present in truffle-config when I branched off. I don't remember who added it here.
But I can change that to read values from `.secrets.json`

**Comment by Rishabh42** [comment]
@jchittoda made the changes, please review
Thanks

File: migrations/2_deploy.js [hunk 4]
```diff
@@ -1,4 +1,4 @@
-var tokenData = require("../test-helpers/tokenData.json");
+var tokenData = require("../test-helpers/PolygonDeploymentData.json");
```
Comment (jchittoda) [inline migrations/2_deploy.js:1]: We should keep using `tokenData.json` in the migration scripts. As these scripts are used in the test coverage. So we need to use the consistent data.

**Review by jchittoda — COMMENTED** [review]
Please fix the one last comment. Then this PR is ready for merge.

File: migrations/2_deploy.js (hunk 4 above)
Comment (Rishabh42) [reply migrations/2_deploy.js:1]: sure

**Review by jchittoda — APPROVED** [review]

**Pull request merged**